app/views/layouts/base.html.erb

```diff
@@ -69,13 +70,13 @@
         <% end %>
       <% end %>
     </div>
-    <div class="container">
+    <main class="container" id="main-content">
```

Comment:
I know that this doesn't change anything for the end user. Just wanted to start using more semantic HTML elements.

Reply:
> I know that this doesn't change anything for the end user. Just wanted to start using more semantic HTML elements.

I like this idea 👍  I see that the accessibility tests didn't take so kindly to it, though, due to the `site-wrap` wrapper 😕  Maybe it's best to implement this separately, or else I wonder if we could remove the wrapper by moving its styles elsewhere.

Reply:
@aduth I am thinking that we implement this separately. I am going to fiddle around with this and see what happens.